Garmin.Connect/Models/GarminWorkout.cs

```diff
@@ -0,0 +1,145 @@
+using System;
+using System.Text.Json.Serialization;
+using Garmin.Connect.Converters;
+
+namespace Garmin.Connect.Models;
+
+public class GarminWorkout
```

Comment:
Is it expected to have GarminWorkout as class, not a record? It means I can not change the workout name or some other fields...

Reply:
Oh sorry I missed that I thought all ok and merged
I will fix this in the next release with calendar methods.